docs/HACKING.md

```diff
@@ -1,6 +1,29 @@
 ⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
 
 # Hacking on the WMCO
+Requires all pre-requisites in the [README](/README.md) to be fufilled.
+
+## User Workflows
+
+### I want to install an officially released version of WMCO on my OpenShift/OKD cluster
+* Instructions to install the community operator can be found in the [README](/README.md).
+  Paying customers should instead look at the [OpenShift docs.](https://docs.openshift.com/container-platform/latest/windows_containers/enabling-windows-container-workloads.html)
+
+### I want to try a specific commit of WMCO
+* [Build the operator](#build)
+* [Deploy the operator directly](#deploying-the-operator-directly)
+
+### I want to try a specific commit of WMCO, and deploy it using OLM
+* [Build the operator](#build)
+* [Deploy the operator through OLM](#deploying-the-operator-through-olm)
+
+### I want to deploy a specific commit of WMCO, in a way that is as close to the official deployment method as possible
+* [Build the operator](#build)
+* [Generate bundle manifests](#generating-new-bundle-manifests)
+* [Build the bundle image](#creating-a-bundle-image)
+* [Build an operator index](#creating-a-new-operator-index)
+* [Create a CatalogSource using the index](https://docs.openshift.com/container-platform/4.7/operators/admin/olm-managing-custom-catalogs.html#olm-creating-catalog-from-index_olm-managing-custom-catalogs)
```

Comment:
Replacing the `4.7` in the URL with `latest` should keep us pointed to the most up-to-date official docs.